web3/contract/base_contract.py

```diff
@@ -956,6 +1121,26 @@ def get_function_by_identifier(
             "This method should be implemented in the inherited class"
         )
 
+    @combomethod
+    def find_events_by_identifier(
```

Comment:
nit:`find/get_functions/events_by_id` are all beneath the `# Private helpers` comment. Consider moving above 🤷